### ISSUE TITLE
Add @see phpDoc annotation in generated views

### ISSUE THREAD

**Issue by naffiq** [yii2tech/admin#15]
It would be very useful for IDE users if generated views contained corresponding controller actions, like following 

```php
<?php

// Corresponding controller and action
/** @see \app\modules\admin\feedback\controllers\DefaultController corresponding controller*/
/** @see \yii2tech\admin\actions\Index corresponding action */

use yii\grid\GridView;
use yii2tech\admin\grid\ActionColumn;

/* @var $this yii\web\View */
/* @var $searchModel app\models\search\FeedbackSearch */
/* @var $dataProvider yii\data\ActiveDataProvider */

$this->title = 'Feedbacks';
$this->params['breadcrumbs'][] = $this->title;
$this->params['contextMenuItems'] = [
    ['create']
];
?>

<?= GridView::widget([
    'dataProvider' => $dataProvider,
    'filterModel' => $searchModel,
    'columns' => [
        ['class' => 'yii\grid\SerialColumn'],
        'id',
        'title',
        'phone',
        'email:email',
        'body:ntext',
        [
            'class' => ActionColumn::className(),
        ],
    ],
]); ?>
```

It might be too much for some users so this feature could be configurable via checkbox or something like that.

**Comment by klimov-paul** [comment]
Resolved by commit c6220c91013acc440cc0e346eefa6791053b2046

**Comment by naffiq** [comment]
@klimov-paul cool, didn't expect it so fast! :D
